src/platform_impl/windows/window.rs

```diff
@@ -794,7 +794,8 @@ impl<'a, T: 'static> InitData<'a, T> {
         let window_state = {
             let window_state = WindowState::new(
                 &self.attributes,
-                self.pl_attribs.taskbar_icon.clone(),
+                // Will be set later in on_create
+                None,
```

Comment:
let's just remove it from the constructor then as this is the only place we call `WindowState::new`

Reply:
sure